### PR TITLE
Add list of log sources to log vision

### PR DESCRIPTION
**Pull request by PettitWesley**
I created this list by looking at the list of log inputs for Fluentd, Fluent Bit, and Microsoft's recently announced [Container Log Monitor](https://techcommunity.microsoft.com/t5/containers/windows-containers-log-monitor-opensource-release/ba-p/973947#).

### CODE REVIEW

File: text/0092-logs-vision.md [hunk 1]
```diff
@@ -96,6 +96,16 @@ This is important because we believe software that was created before
 OpenTelemetry should not be disregarded and should benefit from OpenTelemetry
 efforts where possible.
 
+Specifically, we should be able to collect the following sources:
+- Log files, especially those emitted by container runtimes
+- Syslog
+- Systemd/Journald
+- Windows Event Log
+- Event Tracing for Windows (ETW)
+- JSON over TCP or UDP
```
Comment (PettitWesley) [inline text/0092-logs-vision.md:105]: We use raw JSON over TCP at AWS with some libraries: https://github.com/awslabs/aws-embedded-metrics-python

Anecdotally, having worked with Fluentd and Fluent Bit for 2 years, these inputs are rare/niche compared to others, but definitely in use by a good number of people.

File: text/0092-logs-vision.md [hunk 2]
```diff
@@ -96,6 +96,16 @@ This is important because we believe software that was created before
 OpenTelemetry should not be disregarded and should benefit from OpenTelemetry
 efforts where possible.
 
+Specifically, we should be able to collect the following sources:
+- Log files, especially those emitted by container runtimes
+- Syslog
+- Systemd/Journald
+- Windows Event Log
```
Comment (PettitWesley) [inline text/0092-logs-vision.md:103]: I think these four are obvious and won't lead to any disagreement, the others were added for completeness, but I'm happy to remove them if thats desired

Comment (tigrannajaryan) [reply text/0092-logs-vision.md:103]: I agree, these look like basic things that we do want to support. We may need to discuss and decide the priorities.

These are requirements for the Collector, right? We do not intend to collect these using e.g. OpenTelemetry SDK. If these are for the Collector then best to make these Collector roadmap items.

File: text/0092-logs-vision.md (hunk 1 above)
Comment (tigrannajaryan) [reply text/0092-logs-vision.md:106]: These are probably good fit for the Contrib. I do not think they should be part of the vision or core roadmap. Collector currently encourages to actively contribute components that support specific formats to Contrib repo and we already see this happening for traces and metrics. I believe we need to continue the practice for logs. If anyone is interested in these particular formats they should be able to add these as components.

**Pull request closed without merging**